libraries/AP_Landing/AP_Landing_Slope.cpp

```diff
@@ -349,9 +349,7 @@ int32_t AP_Landing::type_slope_get_target_airspeed_cm(void)
 
     switch (type_slope_stage) {
```

Comment:
Changing from cruise to half way between min and cruise if land airspeed is not set.